index.html

```diff
@@ -3514,6 +3514,81 @@ <h3>Reserved Extension Points</h3>
         </p>
 
       </section>
+
+      <section class="normative">
+        <h3>Ecosystem Compatibility</h3>
+
+        <p>
+There are a number of digital credential formats that do not natively use the
+data model provided in this document, but are aligned with a number of concepts
+in this specification. At the time of publication, examples of these digital
+credential formats include
+<a href="https://www.rfc-editor.org/rfc/rfc7519.html">
+JSON Web Tokens</a> (JWTs),
+<a href="https://www.rfc-editor.org/rfc/rfc8392.html">
+CBOR Web Tokens</a> (CWTs),
+<a href="https://www.iso.org/standard/69084.html">ISO-18013-5:2021</a>
+(mDLs),
+<a href="https://hyperledger.github.io/anoncreds-spec/">
+AnonCreds</a>,
+<a href="https://www.ietf.org/archive/id/draft-mcnally-envelope-02.html">
+Gordian Envelopes</a>, and
+<a href="https://www.ietf.org/archive/id/draft-ssmith-acdc-02.html">
+Authentic Chained Data Containers</a> (ACDCs).
+        </p>
+
+        <p>
+If conceptually aligned digital credential formats can be transformed into a
+<a>conforming document</a> according to the rules provided in this section, they
+are considered <em>"compatible with the Verifiable Credentials ecosystem"</em>.
+A <a>conforming document</a> is either a <a>verifiable credential</a> serialized
+as the `application/vc+ld+json` media type or a <a>verifiable presentation</a>
+serialized as the `application/vp+ld+json` media type. Specifications that are
+compatible with the Verifiable Credentials ecosystem:
+        </p>
+
+        <ul>
+          <li>
+MUST identify whether the transformation to this data model is one-way-only or
+round-trippable.
+          </li>
+          <li>
+MUST preserve the `@context` values when performing round-trippable
+transformation.
+          </li>
+          <li>
+MUST result in a <a>conforming document</a> when transforming to the data
+model described by this specification.
+          </li>
+          <li>
+MUST specify a registered media type for the input document.
+          </li>
+          <li>
+SHOULD provide a test suite that demonstrates that the specified transformation
+algorithm to the data model in this specification results in
+a <a>conforming document</a>.
+          </li>
+          <li>
+SHOULD ensure that all semantics utilized in the transformed
+<a>conforming document</a> follow best practices for Linked Data. See
+Section <a href="#getting-started"></a>, Section
+<a href="#extensibility"></a>, and Linked Data Best Practices [[?LD-BP]]
+for additional guidance.
+          </li>
+        </ul>
+
+        <p class="note" title="What constitutes a verifiable credential?">
+Readers are advised that a digital credential is only considered a
+<a>verifiable credential</a> or a <a>verifiable presentation</a> if it is
+transformed into a <a>conforming document</a>, as described by this
```

Comment:
This potentially confuses transforming and securing.

For example in v1.0 and v1.1, we considered "vc-jwt" a "verifiable credential" with external proof... And it required a mapping... We called the unmapped thing a verifiable credential.

OIDC calls mDoc verifiable credential, and they are unmapped...

Are we saying that a Verifiable Credential is only a JSON-LD document? 

If so, no external proof dependent media type (like jwt, jades or SD-JWT) can be considered a Verifiable Credential... They can only be considered securing formats that are "transformable to Verifiable Credentials".... 


I think it's a mistake to change the definition of verifiable credential to be only a JSON-LD document... It is better that we allow w3c verifiable credentials to be any content type that can secure a JSON-LD document.

I object to the note, but I approve all the other changes... I think we can probably find language for the note that will work.

Reply:
Yes, good points, I'll try to come up with language that strikes that balance... the intent of the note wasn't to say that an `application/vc+ld+json` secured via JOSE/COSE/SD-JWT /was not/ a VC. I agree, those are VCs... we just didn't want someone proclaiming that a bare JWT w/ no transform to a conforming document per this specification was a verifiable credential.

Reply:
I have attempted a clarification in c516f2c9a0282ca59397341f89c61c60b98fb02d that makes it clear that external securing formats that secure a conforming document are considered verifiable credentials.